package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@square/web-sdk",
-  "version": "2.0.0-alpha.2",
```

Comment:
🤔  how did you update in #74 and for this PR? I was expecting a smaller diff

Reply:
`npm upgrade @square/web-payments-sdk-types`

`web-payments-sdk-types` added typescript as a dependency to ensure the proper version was added, but I don't think it is related to this.